src/components/TextInput/index.jsx

```diff
@@ -1,7 +1,46 @@
-import React from "react";
+import React, { useState } from "react";
+import styles from "./styles/styles.module.scss";
 
-const TextInput = () => {
-  return <p>text input</p>;
+const TextInput = ({ label, minlength, isRequired = true }) => {
```

Comment:
```js
```

Reply:
Please use camel case format for variables name.

Reply:
done